storage/engine/rocksdb/db.cc

```diff
@@ -1089,6 +1130,8 @@ DBStatus DBOpen(DBEngine **db, DBSlice dir, DBOptions db_opts) {
     table_options.block_cache = rocksdb::NewLRUCache(
         block_cache_size, num_cache_shard_bits);
   }
+  table_options.filter_policy.reset(rocksdb::NewBloomFilterPolicy(10));
+  table_options.format_version = 2;
```

Comment:
No, changing the format did nothing. It isn't even necessary for prefix-based bloom filters. This was a drive-by changed based on the following comment in the rocksdb source:

```
  // We currently have three versions:
  // 0 -- This version is currently written out by all RocksDB's versions by
  // default.  Can be read by really old RocksDB's. Doesn't support changing
  // checksum (default is CRC32).
  // 1 -- Can be read by RocksDB's versions since 3.0. Supports non-default
  // checksum, like xxHash. It is written by RocksDB when
  // BlockBasedTableOptions::checksum is something other than kCRC32c. (version
  // 0 is silently upconverted)
  // 2 -- Can be read by RocksDB's versions since 3.10. Changes the way we
  // encode compressed blocks with LZ4, BZip2 and Zlib compression. If you
  // don't plan to run RocksDB before version 3.10, you should probably use
  // this.
  // This option only affects newly written tables. When reading exising tables,
  // the information about version is read from the footer.
```

I did benchmark this change in isolation and it did not result in any performance change.